### ISSUE TITLE
High FPS causes vehicles to auto brake

### ISSUE THREAD

**Issue by ADRFranklin** [Whitetigerswt/gtasa_crashfix#57]
After installing Crashes 2.4, I noticed that when I turn off frame limiter, vehicles begin to brake when you stop accelerating.

This causes some unexpected driving, and is really just a major annoyance. I assume it has something to do with the way gravity/resistance is calculated per frame. (just a guess)

**Comment by Whitetigerswt** [comment]
Yeah, this happens in vice city, and gta 3. I swear I was playing gta V the other day and thought I saw it happen. Not really sure if this is something that is easy to fix. It only happens above 100 fps I think

**Comment by ADRFranklin** [comment]
@Whitetigerswt https://gtaforums.com/topic/760017-samp-addon/
I found this and tried it out, and it seems similar to this mod, however it has it's own couple of issues. yet somehow it doesn't seem to have the issue with high FPS affecting the braking or most mechanics of the game.

Is this something you could comb over and maybe try implementing here?

**Comment by RobbyAJM** [comment]
For those who still facing this issue. Use this cleo to fix

https://stringer.ro/carspeedfpsfix/
